frontend/public/components/cloud-services/install-plan.tsx

```diff
@@ -34,7 +34,7 @@ export const InstallPlanRow: React.SFC<InstallPlanRowProps> = (props) => {
       <ResourceLink kind="Namespace" name={props.obj.metadata.namespace} title={props.obj.metadata.namespace} displayName={props.obj.metadata.namespace} />
     </div>
     <div className="hidden-xs col-sm-4 col-md-3 col-lg-2">
-      {props.obj.spec.clusterServiceVersionNames.map((csvName, i) => <span key={i}><ResourceIcon kind={referenceForModel(ClusterServiceVersionModel)} /> {csvName}</span>)}
+      {props.obj.spec.clusterServiceVersionNames.map((csvName, i) => <span key={i}><ResourceLink kind={referenceForModel(ClusterServiceVersionModel)} name={csvName} namespace={props.obj.metadata.namespace} /></span>)}
```

Comment:
Use the same [logic as in `InstallPlanDetails`](https://github.com/openshift/console/blob/master/frontend/public/components/cloud-services/install-plan.tsx#L88) to make sure we don't link to a `ClusterServiceVersion` that doesn't exist.

Reply:
Requested changes updated